### PR TITLE
Best sellers frontend

### PR DESCRIPTION
**Pull request by julkascript**
Closes #83 

image

image

image

### CODE REVIEW

**Review by milensski — APPROVED** [review]

**Review by dinocom33 — APPROVED** [review]

**Comment by dinocom33** [comment]
There is no direct link/button to "buy" page. Buy button leads to "search" page.

**Comment by RyotaMitaraiWeb** [comment]
On certain viewports, the page looks like this:
incorrect_alignment

On slightly larger viewports, the page also looks like this:
incorrect_alignment2

I have included the widths in the screenshot

File: frontend/src/pages/yugioh/BestSellers.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  return (
+    <section className="bg-[#F5F5F5] min-h-[100vh]">
+      <ListingTopBar />
+      <PageHeader heading="Buy" />
+      <div className="w-5/6 mx-auto my-4">
+        <PageSection className="p-8 my-4">
+          <Typography variant="h4">All-time best sellers</Typography>
+          <hr />
+          <section className="flex max-sm:flex-col gap-4 mt-6 flex justify-between">
```
Comment (RyotaMitaraiWeb) [inline frontend/src/pages/yugioh/BestSellers.tsx:65]: ``flex`` is used twice on the subsection

**Review by RyotaMitaraiWeb — APPROVED** [review]
Looks good to me

**Pull request merged**